pyproject.toml

```diff
@@ -1,5 +1,123 @@
 [build-system]
-requires = ["setuptools >= 38.0.0",
-            "setuptools_scm >= 2.0.0",
-            "wheel"]
-build-backend = 'setuptools.build_meta'
\ No newline at end of file
+requires = ["setuptools >= 61.2",
+            "setuptools_scm[toml]>=7.1"]
+build-backend = 'setuptools.build_meta'
+
+[project]
+name = "poppy"
+description = "Physical optics propagation (wavefront diffraction) for optical simulations, particularly of telescopes."
+authors = [{name = "Marshall Perrin", email = "mperrin@stsci.edu"}]
+license = {text = "BSD-3-Clause"}
+requires-python = ">=3.9"
+dependencies = [
+    "numpy>=1.20.0",
+    "scipy>=1.5.0",
+    "matplotlib>=3.2.0",
+    "astropy>=4.3.0",
+]
+dynamic = ["version"]
```

Comment:
I went through some of the example notebooks and noticed that `import poppy; poppy.__version__` gives an empty string. Working backwards, it appears this has been the case since at least v1.0.3, so you didn't introduce that here.

I do see that a `poppy/version.py` file is generated as part of the installation process, so that's working OK. Is there a simple fix to be made here that can also bring back the `__version__` attribute?

Reply:
that is done here:
https://github.com/spacetelescope/poppy/blob/62e94c43c657ede92b5e840abad6eaff2fe1bce1/poppy/__init__.py#L21-L24
and it looks like the `version.py` looks like this:
```
# file generated by setuptools_scm
# don't change, don't track in version control
__version__ = version = '1.0.4.dev100+ged0a8b1.d20230501'
__version_tuple__ = version_tuple = (1, 0, 4, 'dev100', 'ged0a8b1.d20230501')
```
I'm not sure what's going wrong to cause the import to fail

Reply:
ah, of course; it should be `from .version import __version__`

Reply:
Good catch all, thanks for the input.  Just pushed the change